scripts/scripts/testing/automations/batch/lebanon.py

```diff
@@ -0,0 +1,21 @@
+import pandas as pd
+from bs4 import BeautifulSoup
+import requests
+from datetime import date
+
+url = 'https://corona.ministryinfo.gov.lb'
+req = requests.get(url)
+soup = BeautifulSoup(req.text, 'html.parser')
+
+count = int(data.find_all('h1',attrs={'class':'s-counter-3 s-counter'}).text)

```

Comment:
Variable `data` is not defined.

I tried substituting for `soup` but getting an error:

```
raise AttributeError(
AttributeError: ResultSet object has no attribute 'text'. You're probably treating a list of elements like a single element. Did you call find_all() when you meant to call find()?
```

Tried to use the method `find` instead of `find_all` as suggested by error message but still getting an error:

```
count = int(soup.find('h1',attrs={'class':'s-counter-3 s-counter'}).text)
AttributeError: 'NoneType' object has no attribute 'text'
```